### ISSUE TITLE
Checkbox onChange handler does not supply field information

### ISSUE THREAD

**Issue by MagicLegend** [scandipwa/scandipwa#4296]
**Description**:
Checkbox `onChange` handler does not supply the field information like the regular text input type does. 

**Expected behavior**:
For the checkbox `onChange` handler does to supply the field information like the regular text input type does. 

**Screenshots, Video, Logs**
If applicable, add screenshots or screen-capture to help explain your problem. If you would like to share logs - please use [fenced code blocks](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks).

**To Reproduce**:
```
        {
            type: FIELD_TYPE.checkbox,
            label: __('Save?'),
            attr: {
                name: 'shouldSave',
                defaultValue: false,
                value: currShouldSave
            },
            events: {
                onChange: onShouldSaveChange
            }
        }
```

```
    onShouldSaveChange(event, field) {
        const { value } = field;

        this.setState({ currShouldSave: value });
    }
```

`field` will be `undefined` here.

**Versions**:
- ScandiPWA: 5.2.4

**Comment by AleksandrsKondratjevs** [comment]
Well, won't event.target.value work for you?

**Comment by MagicLegend** [comment]
No, I am lazy :wink:  It's just inconsistent with the other field types, hence filing it as a bug report. I'd expect this to work :smile:
